src/appleseed-max-impl/utilities.h

```diff
@@ -377,23 +377,23 @@ const T load_system_setting(const wchar_t* key_name, const T& default_value)
 }
 
 template <typename T>
-void save_ini_setting(const T& value, const wchar_t* category, const wchar_t* key_name)
+void save_ini_setting(const wchar_t* category, const wchar_t* key_name, const T& value)
```

Comment:
You need to update the forward declarations of these functions too.

Reply:
Terrible mistake!